samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/main.c

```diff
@@ -198,17 +197,17 @@ void main(void)
 	ps_settings_init();
 
 	/* Initialize the Bluetooth Subsystem */
-	err = bt_enable(bt_ready);
+	err = bt_enable(NULL);
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
```

Comment:
@vikrant8051 you had earlier a return statement here. Why did you remove it? I suppose none of the subsequent code in main() makes sense if bt_enable() fails. Also, if/when you add back the return statement, the else branch can be removed (i.e. call bt_ready on the same level as e.g. light_default_status_init)

Reply:
Done !!